docs/setup_toolchain.md

```diff
@@ -53,8 +53,8 @@ Next step: [Add keyboard support to Arduino](#add-keyboard-support-to-arduino)
 3. On Ubuntu, you will have to prevent ModemManager from attempting to grab the keyboard virtual serial port when it goes into bootloader mode:
```

Comment:
It's not just Ubuntu, and the rules now do more than just inhibiting ModemManager.

Reply:
Updated the docs. thanks.